src/main/app.js

```diff
@@ -15,11 +17,16 @@ app.on('window-all-closed', () => {
 
 app.on('ready', () => {
   let main = new BrowserWindow({width: 800, height: 600})
+  let { webContents } = main
+
+  const navigate  = sendToChannel(webContents, 'navigate')
+
   main.loadURL(`file://${html}`)
 
-  Menu.setApplicationMenu(Menu.buildFromTemplate(menus()))
+  Menu.setApplicationMenu(Menu.buildFromTemplate(menus({ navigate })))
 
   main.on('closed', () => {
+    webContents = null
```

Comment:
No need for this anymore, but the is a potential memory :droplet: with the Menu due to the `main.webContents` hook.
A good :wrench: would be to clear up the reference inside of the Application Menu by doing a little:
```javascript
Menu.setApplicationMenu(null)
```
in the `main` closed handler.